### ISSUE TITLE
<img> attributes are being transfered to the <source> tag, rendering invalid HTML

### ISSUE THREAD

**Issue by hd-jeroen** [yireo/Yireo_NextGenImages#78]
I have created an `<img>` tag with the attributes `alt`, `decoding`, `fetchpriority` and `loading`. When NextGenImages converts the image to the WebP all of these attributes are being transfered over to the `<source>` element which holds the WebP image:

```
<picture>
    <source type="image/webp" srcset="https://www.ecobouwbescherming.nl/media/wysiwyg/categories/meubellakken.webp" alt="Meubellakken" decoding="async" fetchpriority="high" loading="eager">
    <img src="https://www.ecobouwbescherming.nl/media/wysiwyg/categories/meubellakken.jpg" alt="Meubellakken" decoding="async" fetchpriority="high" loading="eager">
</picture>
```

As per the HTML spec, this isn't allowed. The `<source>` HTML element only allows the `media`, `sizes`, `src`, `srcset` and `type` attributes. See below screenshot from the W3 HTML validator.

![image](https://github.com/user-attachments/assets/61a8866a-1941-40ac-89c7-d2b39fc261b1)

Would it be possible to change the NextGenImages implementation to filter out all invalid HTML attributes, thus making the HTML valid again?

**Comment by jissereitsma** [comment]
Good point: I think the following fix will repair this: https://github.com/yireo/Yireo_NextGenImages/commit/c3ad533a2d1b05a31a4375632cc93b828f0c4592 (to be released today)
